Index: ../010.Quiz_game/main.py
===================================================================
diff --git a/../010.Quiz_game/main.py b/../010.Quiz_game/main.py
--- a/../010.Quiz_game/main.py	
+++ b/../010.Quiz_game/main.py	
@@ -1,0 +1,21 @@
+from question_model import Question
+from data import question_data
+from quiz_brain import QuizBrain
+
+question_bank = []
+
+for question in question_data:
+    question_text = question["text"]
+    question_answer = question["answer"]
+
+    new_question = Question(question_text, question_answer)
+    question_bank.append(new_question)
+
+quiz = QuizBrain(question_bank)
+
+while quiz.still_has_questions(): #if quiz still has questions remaining
+    quiz.next_question()
+
+print("You've completed the quiz.")
+print(f"Your final score was: {quiz.user_score}/{quiz.question_number}")
+
